mono/mini/mini-runtime.c

```diff
@@ -3395,7 +3399,11 @@ MONO_SIG_HANDLER_FUNC (, mono_sigsegv_signal_handler)
 		if (!ji && mono_chain_signal (MONO_SIG_HANDLER_PARAMS))
 			return;
 
-		mono_arch_handle_altstack_exception (ctx, info, info->si_addr, FALSE);
+		if (info->si_addr > UPPER_BOUND_FOR_IMPLICIT_NULL_REFERENCE_EXCEPTION) {
```

Comment:
I'm not sure having this as a define is a good idea since there's a function call behind it.